data_samples/conll03_new/conll03.conll

```diff
@@ -0,0 +1,9 @@
+EU NNP B-NP B-ORG
+rejects VBZ B-VP O
+German JJ B-NP B-MISC
+call NN I-NP O
+to TO B-VP O
+boycott VB I-VP O
+British JJ B-NP B-MISC
+lamb NN I-NP O
+. . O O
```

Comment:
I think we need a little bit more complex testing sample, which tests the following cases:

* BIO tag spaned over multiple tokens. For example:
```
B-ORG
I-ORG
I-ORG
O
```

* Two continuing BIO tags. For example:
```
B-ORG
I-ORG
I-ORG
B-MISC
I-MISC
O
```

Reply:
Got it.

Reply:
I have add a more complex case, where BIO spans for multiple tokens.